<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="BRIMACLAP042.BRILLIO.COM" failures="1" tests="2" name="Test" time="0.961" errors="0" timestamp="23 Jul 2019 04:47:12 GMT">
  <testcase classname="howToDoInJava.testNg.TimeoutMethod" name="timeTestOne" time="0.52">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method howToDoInJava.testNg.TimeoutMethod.timeTestOne() didn&amp;apos;t finish within the time-out 500">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method howToDoInJava.testNg.TimeoutMethod.timeTestOne() didn't finish within the time-out 500
at java.base@10.0.1/java.lang.Thread.interrupt(Thread.java:1016)
at java.base@10.0.1/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:514)
at java.base@10.0.1/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base@10.0.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1135)
at java.base@10.0.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base@10.0.1/java.lang.Thread.run(Thread.java:844)
... Removed 2 stack frames]]>
    </failure>
  </testcase> <!-- timeTestOne -->
  <testcase classname="howToDoInJava.testNg.TimeoutMethod" name="timeTestTwo" time="0.412"/>
</testsuite> <!-- Test -->
